qt/scientific_interfaces/ISISReflectometry/Reduction/LookupTable.cpp

```diff
@@ -44,7 +47,15 @@ boost::optional<LookupRow> LookupTable::findLookupRow(Row const &row, double tol
   }
   // If we didn't find a lookup row where theta matches, then we allow the user to specify a "wildcard" row
   // which will be used for everything where a specific match is not found
-  return findWildcardLookupRow();
+  auto result = findWildcardLookupRow();
+  if (result) {
+    g_log.warning("Used wildcard row for " + boost::algorithm::join(row.runNumbers(), ", ") +
```

Comment:
Could we mention this relates to the experiment settings? Also the warning is before the algorithm runs so I'd say "using" rather than "used". 

I think it's fine to be a bit wordy here for clarity so maybe something like `No matching experiment settings were found for <ws>. Settings will be taken from the wildcard row instead. You may wish to check that any lookup criteria on the Experiment Settings table are correct.`